### PR TITLE
EZP-28727: Edit button from Multi-file upload does not redirect to Edit form

### PR DESCRIPTION
**Pull request by sunpietro**
https://jira.ez.no/browse/EZP-28727

### CODE REVIEW

File: src/modules/multi-file-upload/components/upload-list/upload.item.component.js [hunk 1]
```diff
@@ -309,12 +324,19 @@ export default class UploadItemComponent extends Component {
...
+
+        if (!struct) {
             return;
         }
 
-        return Routing.generate('ez_content_draft_create', {
-            contentId: this.state.struct.Content._id,
+        const content = struct.Content;
+
+        return window.Routing.generate('ez_content_draft_edit', {
```
Comment (dew326) [inline src/modules/multi-file-upload/components/upload-list/upload.item.component.js]: This should not be done by edit link but should be sent a form to edit. Should be done like is in SubItemList. // cc @webhdx

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/modules/multi-file-upload/components/upload-list/upload.item.component.js (hunk 1 above)
Comment (sunpietro) [reply src/modules/multi-file-upload/components/upload-list/upload.item.component.js]: Good catch! :)

**Comment by sunpietro** [comment]
ping @dew326

**Review by dew326 — APPROVED** [review]

**Review by mnocon — APPROVED** [review]
Looks good!

**Pull request merged**